tests/test_E3_indicator_calculation.py

```diff
@@ -663,6 +664,47 @@ def test_equation_degree_of_autonomy():
     )
 
 
+def test_add_degree_of_net_zero_energy():
+    """ """
+    total_re_generation = 60
+    total_demand = 100
+    excess = 10
+    dict_values_NZE = {
+        KPI: {
+            KPI_SCALARS_DICT: {
+                TOTAL_RENEWABLE_GENERATION_IN_LES: total_re_generation,
+                TOTAL_DEMAND + SUFFIX_ELECTRICITY_EQUIVALENT: total_demand,
+                TOTAL_EXCESS + SUFFIX_ELECTRICITY_EQUIVALENT: excess,
+            }
+        },
+    }
+    E3.add_degree_of_net_zero_energy(dict_values_NZE)
+
+    degree_of_nze = (total_re_generation - excess) / total_demand
+
+    assert (
+        DEGREE_OF_NZE in dict_values_NZE[KPI][KPI_SCALARS_DICT]
+    ), f"The {DEGREE_OF_NZE} is not added to {KPI_SCALARS_DICT}"
+    assert (
+        dict_values_NZE[KPI][KPI_SCALARS_DICT][DEGREE_OF_NZE] == degree_of_nze
+    ), f"The degree of NZE is not added successfully to the list of KPI's."
+
+
+def test_equation_degree_of_net_zero_energy():
```

Comment:
You could have tested the three cases > 1, <1, =1 here, to make sure that our expectations regarding input-output-transformation are correct.

Reply:
If the  formula works for one case it should work for the others too, except if there can be negative values in the input parameters, which is not the case.